libs/vscode/nx-project-view/src/lib/helper.ts

```diff
@@ -0,0 +1,48 @@
+import { ProjectConfiguration } from "@nrwl/devkit";
+
+export class NxProjectTreeHelper {
+
+  static async getProjectsInFolders(projects: [string, ProjectConfiguration][]): Promise<Record<string,ProjectConfiguration>> {
+    const projectsObject = {};
+
+    projects.forEach((project) => {
+      const lastArray = project[1] as ProjectConfiguration;
+      const splittedProject = lastArray.root.split('/');
+
+      splittedProject.reduce((previousValue: {[key: string]: string;} & any, currentValue: string, currentIndex: number) => {
+          if (splittedProject.length -1 > currentIndex) {
+              return previousValue[currentValue] || (previousValue[currentValue] = {});
+          }
+          return previousValue[currentValue] || (previousValue[currentValue] = project)
+      }, projectsObject)
+    });
+
+    return projectsObject;
+  }
+
+  static async findNestedObject(fields: any, key: string): Promise<any> {
```

Comment:
I was hoping that with the change of the type in the previous function, that you would remove the `any` types from here as well 😅 

Please make sure all new code are properly typed please 🙂